### DISCUSSION TITLE
Is it possible to use other modles for OCR?

### DISCUSSION THREAD

**Discussion by fernvenue** [Stirling-Tools/Stirling-PDF#1217] [Q&A] [answered]
Hi all, I just wanna know if we can use other modles for OCR so that we can get better experience when we use OCR for other languages.

**Comment by Frooodle** [comment, marked as answer]
not currently sorry only OCRMyPDF (tessract) is currently implemented

**Reply by fernvenue** [reply]
Hi! Thanks for reply, much appreciate. And the reason why I ask this question is because I think [PaddlePaddle/Paddle](https://github.com/PaddlePaddle/Paddle) have a really good performance in multiple languages, that might be a good alternative :)
